src/components/tabs/chapters/ChapterEntry.vue

```diff
@@ -193,11 +212,26 @@
       width: calc(100% - #{$index-width});
 
       .title {
-        width: calc(100% - #{$index-width});
+        width: calc(100% - 4.4em);
         pointer-events: none;
       }
 
+      .icon {
+        flex-shrink: 0;    
+      }
+          
+      .info-link {
+        font-weight: 700;
+        text-align:right;
```

Comment:
there is a space missing

Reply:
Ah, something the linter didn't see. BTW: is there some invocation of yarn lint that actually just fixes the problems it sees? I was too blind to see it if it exists.